loleaflet/src/control/Control.JSDialogBuilder.js

```diff
@@ -1347,9 +1347,8 @@ L.Control.JSDialogBuilder = L.Control.extend({
 			image.src = data.image;
```

Comment:
this image inserted into pushbutton will be overwritten by innerHTML call later I think

Reply:
also please check if sidebar popups which have buttons with images are correctly shown